src/lib/snarky_group_map/snarky_group_map.mli

```diff
@@ -2,7 +2,10 @@ module Params : sig
   type 'f t
 
   val create :
-    (module Group_map.Field_intf.S with type t = 'f) -> a:'f -> b:'f -> 'f t
+       (module Group_map.Field_intf.S_unchecked with type t = 'f)
```

Comment:
What's the reason this changed from `Group_map.Field_intf.S` to `Group_map.Field_intf.S_unchecked` ?

Reply:
It now uses `equal : t -> t -> bool`, which is only available for the "unchecked" version.